EventFilter/CTPPSRawToDigi/python/ctppsRawToDigi_cff.py

```diff
@@ -113,6 +113,11 @@
 from EventFilter.CTPPSRawToDigi.ctppsPixelDigis_cfi import ctppsPixelDigis
 ctppsPixelDigis.inputLabel = cms.InputTag("rawDataCollector")
 
+from Configuration.Eras.Modifier_ctpps_2016_cff import ctpps_2016
+from Configuration.Eras.Modifier_ctpps_2017_cff import ctpps_2017
+from Configuration.Eras.Modifier_ctpps_2018_cff import ctpps_2018
+(ctpps_2016 | ctpps_2017 | ctpps_2018).toModify(ctppsPixelDigis, isRun3 = cms.bool(False) )
```

Comment:
Slava pointed to me that the type does not need (should not) be specified here, sorry for missing it earlier.
```suggestion
(ctpps_2016 | ctpps_2017 | ctpps_2018).toModify(ctppsPixelDigis, isRun3 = False )
```

Reply:
I missed it too when I re-read the code before submission. Slava never misses! ;-)
Thanks.
BTW, it works fine as well...